### DISCUSSION TITLE
Expanding the Arduino Project to Gaggia Platinum Vision

### DISCUSSION THREAD

**Discussion by EngMarc** [Zer0-bit/gaggiuino#337] [Ideas]
First I wanted say what a great fun way to repurpose some classic espresso equipment!
Many thanks to the fun filled tinkerers and developers.

I have been repairing two Gaggia Platinum Vision machines for a while now with some marginal success. The challenge is that the old power boards and CPU boards are in very short supply and frankly kind of flaky (mostly the power board). 

I'd be very interested in helping to port this Arduino project to the Platinum Vision if anyone else is interested. I currently have two machines (one repaired but marginally working and the other mostly repaired but has a bad valve that I need to pull and replace). 

They have quite a lot of sensors already in the stock machines. So, it seems it might be fairly easy to rework them. The biggest problem is the boiler heat cycle - it relies on some components that seem to fail easily with voltage variations (suspect). I'll include a photo of the power board for grins. And a photo of the CPU board.

Anyway, if there's interest, I'm game. 

![PowerSupply - Front](https://user-images.githubusercontent.com/48916817/203409864-e43d35ae-c54f-4d62-99eb-252a7be3683f.jpg)
![CPU Board Front v02 08 26](https://user-images.githubusercontent.com/48916817/203409982-824bf792-0517-4d85-8917-8c6443163e1d.jpg)

**Comment by EngMarc** [comment]
So, I'm going to work on this project and invite anyone else interested to join in. I'm a geek with programming experience in C/C++, AWK, GAWK, Vbasic, Fortran and Pearl. 

I'm taking the Gaggiuino base and will be expanding it to add support the the Platinum functions:  grinder, wand steam/hot water, prebrew, auto tamping to brew and shot size. It's a tall order but these machines are primo and kind of forgotten. Parts are sketchy to find and there's some powerful base functionality that's just looking for a new controller - voilá the Platinumuino is born. Wish me luck!

**Comment by lcortopassi** [comment]
Dear Marc, seems we have the same chalenge. I've a Gaggia Platinum Vision and it constantly shown issues, mostly with the dump box (display shows it's full when there is nothing inside), than its began to show the wrench asking me to restart machine to fix some problem and finaly the display just flashing for a while and after sometimes a day or two the machine reborn. Seems to me the problem came from dump box or from a lot of sensors who control everihing. May be you can help me to fix it. I found a new power board which seems hard to find.

**Comment by EngMarc** [comment]
Kind of taken a sabbatical from this project as found too many problems with the control boards and sensors. I guess I kind of gave up as I had too many other things in the works. 

I'll be happy to pass on any information or spares though.

**Comment by lcortopassi** [comment]
I lfet it at the technical assistance since I was not in grade to solve a bunch of problems of this coffee machine.

Thanks anyway for your answer.

Regards and good luck on your new works
________________________________
De: EngMarc ***@***.***>
Enviado: quinta-feira, 30 de maio de 2024 08:24
Para: Zer0-bit/gaggiuino ***@***.***>
Cc: lcortopassi ***@***.***>; Comment ***@***.***>
Assunto: Re: [Zer0-bit/gaggiuino] Expanding the Arduino Project to Gaggia Platinum Vision (Discussion #337)


Kind of taken a sabbatical from this project as found too many problems with the control boards and sensors. I guess I kind of gave up as I had too many other things in the works.

I'll be happy to pass on any information or spares though.
